apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -34,26 +34,42 @@ function wpcom_should_limit_global_styles( $blog_id = 0 ) {
 		}
 	}
 
-	// Do not limit Global Styles on sites created before we made it a paid feature (2022-12-15).
-	if ( $blog_id < 213403000 ) {
+	// Do not limit Global Styles on demo sites.
+	if ( wpcom_global_styles_has_blog_sticker( 'theme-demo-site', $blog_id ) ) {
```

Comment:
Non-blocking: Maybe we can move this condition inside of `wpcom_premium_global_styles_is_site_exempt`? 🤔 In the end, it's a site exception condition.

Reply:
I'd rather keep it outside as an exception to the rule, rather than being part of the exemption logic.